TileFormats/Instanced3DModel/README.md

```diff
@@ -220,52 +245,43 @@ var featureTableBinary = Buffer.concat([positionQuantizedBinary, normalUpOct32PB
 
 ## Batch Table
 
-Contains metadata organized by `batchId` that can be used for declarative styling.
-
-See the [Batch Table](../BatchTable/README.md) reference for more information.
+Contains metadata organized by `batchId` that can be used for declarative styling. See the [Batch Table](../BatchTable/README.md) reference for more information.
 
 ## glTF
 
-The glTF asset to be instanced is stored after the feature table and batch table.
+The glTF asset to be instanced is stored after the Feature Table and Batch Table.
 
-[glTF](https://www.khronos.org/gltf) is the runtime asset format for WebGL.  [Binary glTF](https://github.com/KhronosGroup/glTF/tree/master/extensions/Khronos/KHR_binary_glTF) is an extension defining a binary container for glTF.  Instanced 3D Model uses glTF 1.0 with the [KHR_binary_glTF](https://github.com/KhronosGroup/glTF/tree/master/extensions/Khronos/KHR_binary_glTF) extension.
+[glTF](https://www.khronos.org/gltf) is the runtime asset format for WebGL. Instanced 3D Models uses [glTF 2.0](https://github.com/KhronosGroup/glTF/tree/master/specification/2.0) to embed model data.
 
-`header.gltfFormat` determines the format of the glTF field.  When it is `0`, the glTF field is
+The [binary glTF](https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#binary-gltf-layout) immediately follows the feature table and batch table.  It may embed all of its geometry, texture, and animations, or it may refer to external sources for some or all of these data.
 
-* a UTF-8 string, which contains a url to a glTF model.
+`header.gltfFormat` determines the format of the glTF field
 
-When the value of `header.gltfFormat` is `1`, the glTF field is
+* When the value of `header.gltfFormat` is `0`, the glTF field is a UTF-8 string, which contains a uri of the glTF model content.
+* When the value of `header.gltfFormat` is `1`, the glTF field is a binary blob containing binary glTF.
 
-* a binary blob containing binary glTF.
+> Implementation note: Prefer a binary glTF blob to an glTF model data uri.
```

Comment:
I know this note was my suggestion, but I think it can be removed since I doubt in practice many people will think to store the glTF as a data uri. The sentence as it is might just lead to confusion.